Mathlib/RingTheory/HahnSeries/PowerSeries.lean

```diff
@@ -18,6 +18,11 @@ we get the more familiar semiring of formal power series with coefficients in `R
   * `toPowerSeries` the isomorphism from `HahnSeries ℕ R` to `PowerSeries R`.
   * `ofPowerSeries` the inverse, casting a `PowerSeries R` to a `HahnSeries ℕ R`.
 
+## Instances
+  * For `Finite σ`, the instance `NoZeroDivisors (HahnSeries (σ →₀ ℕ) R)`,
+  deduced from the case of `MvPowerSeries`
+  The case of `HahnSeries ℕ R` is taken care of by `instNoZeroDivisors`.
```

Comment:
Please qualify `instNoZeroDivisors`

Reply:
done. Anyway, this instance only appears in another PR. (I messed up the docstrings)